src/cyw43.h

```diff
@@ -336,6 +336,17 @@ int cyw43_wifi_join(cyw43_t *self, size_t ssid_len, const uint8_t *ssid, size_t
  */
 int cyw43_wifi_leave(cyw43_t *self, int itf);
 
+/*!
+ * \brief Get the signal strength (RSSI) of the wifi network
+ *
+ * For STA (client) mode, returns the signal strength or RSSI of the wifi network.
+ * An RSSI value of zero is returned if you call this function before a network is connected.
+ *
+ * \param self the driver state object. This should always be \c &cyw43_state
+ * \return 0 on success
```

Comment:
Should there be a `\param rssi` here too?

Reply:
I added this during the merge.